### PR TITLE
Add tosca tests

### PR DESCRIPTION
**Pull request by petr-hanzl**
This PR adds tosca jenkins tests for `google cloud`

### CODE REVIEW

File: tosca/validate-evmzero.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            }
+        }
+
+        stage('validate') {
+            steps {
+                sh "rm -f *.cpuprofile *.memprofile *.log"
+
+                catchError(buildResult: 'FAILURE', stageResult: 'FAILURE', message: 'Test Suite had a failure') {
+                    sh "build/aida-vm --workers 20 --aida-db /mnt/aida-db-central/aida-db --vm-impl evmzero --validate-tx ${BlockFrom} ${BlockTo}"
+                }
```
Comment (Lubomir-Jahn) [inline tosca/validate-evmzero.jenkinsfile:59]: Do we know how much data will this store in the `/tmp`. Problem is that boot disk where `/tmp` is, has only 20GB of space and will run out of it pretty quickly causing whole job to fail and leaving the VM unusable.

File: tosca/validate-geth.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        stage('Build') {
+            steps {
+                script {
+                    currentBuild.description = "Building on ${env.NODE_NAME}"
+                }
+
+                checkout scmGit(
+                    branches: [[name: "${AidaVersion}"]],
+                    userRemoteConfigs: [[
+                        credentialsId:  'AidaCI_PR',
```
Comment (Lubomir-Jahn) [inline tosca/validate-geth.jenkinsfile:33]:
```suggestion
```

File: tosca/validate-geth.jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                        url: 'https://github.com/Fantom-foundation/Aida.git'
+                    ]]
+                )
+                sh "git submodule update --init --recursive"
+
+                dir('tosca') {
+                	checkout scmGit(
+                		branches: [[name: "${ToscaVersion}"]],
+                		userRemoteConfigs: [[credentialsId:  'AidaCI_PR', url: 'https://github.com/Fantom-foundation/Tosca.git']]
```
Comment (Lubomir-Jahn) [inline tosca/validate-geth.jenkinsfile:42]:
```suggestion
                		userRemoteConfigs: [[url: 'https://github.com/Fantom-foundation/Tosca.git']]
```

File: tosca/validate-lfvm-si.jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        stage('Build') {
+            steps {
+                script {
+                    currentBuild.description = "Building on ${env.NODE_NAME}"
+                }
+
+                checkout scmGit(
+                    branches: [[name: "${AidaVersion}"]],
+                    userRemoteConfigs: [[
+                        credentialsId:  'AidaCI_PR',
```
Comment (Lubomir-Jahn) [inline tosca/validate-lfvm-si.jenkinsfile:33]:
```suggestion
```

File: tosca/validate-lfvm-si.jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+                        url: 'https://github.com/Fantom-foundation/Aida.git'
+                    ]]
+                )
+                sh "git submodule update --init --recursive"
+
+                dir('tosca') {
+                	checkout scmGit(
+                		branches: [[name: "${ToscaVersion}"]],
+                		userRemoteConfigs: [[credentialsId:  'AidaCI_PR', url: 'https://github.com/Fantom-foundation/Tosca.git']]
```
Comment (Lubomir-Jahn) [inline tosca/validate-lfvm-si.jenkinsfile:42]:
```suggestion
                		userRemoteConfigs: [[url: 'https://github.com/Fantom-foundation/Tosca.git']]
```

File: tosca/validate-lfvm.jenkinsfile [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+        stage('Build') {
+            steps {
+                script {
+                    currentBuild.description = "Building on ${env.NODE_NAME}"
+                }
+
+                checkout scmGit(
+                    branches: [[name: "${AidaVersion}"]],
+                    userRemoteConfigs: [[
+                        credentialsId:  'AidaCI_PR',
```
Comment (Lubomir-Jahn) [inline tosca/validate-lfvm.jenkinsfile:33]:
```suggestion
```

File: tosca/validate-lfvm.jenkinsfile [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+                        url: 'https://github.com/Fantom-foundation/Aida.git'
+                    ]]
+                )
+                sh "git submodule update --init --recursive"
+
+                dir('tosca') {
+                	checkout scmGit(
+                		branches: [[name: "${ToscaVersion}"]],
+                		userRemoteConfigs: [[credentialsId:  'AidaCI_PR', url: 'https://github.com/Fantom-foundation/Tosca.git']]
```
Comment (Lubomir-Jahn) [inline tosca/validate-lfvm.jenkinsfile:42]:
```suggestion
                		userRemoteConfigs: [[url: 'https://github.com/Fantom-foundation/Tosca.git']]
```

**Review by Lubomir-Jahn — CHANGES_REQUESTED** [review]

**Comment by Lubomir-Jahn** [comment]
Thanks for your work! Can you please remove the credentials from the build stages?

File: tosca/validate-evmzero.jenkinsfile (hunk 1 above)
Comment (petr-hanzl) [reply tosca/validate-evmzero.jenkinsfile:59]: This tool does not create any `StateDb` on disk. It works with one in memory.

**Review by Lubomir-Jahn — APPROVED** [review]

**Review by Lubomir-Jahn — APPROVED** [review]

**Pull request merged**